xml/admin_nfsganesha.xml

```diff
@@ -119,6 +119,7 @@
 <sect2 xml:id="ceph-nfsganesha-config-service-rgw">
     <title>RGW Section</title>
 <screen>RGW {
+<!-- 2020-09-7 tbazant (ceph.conf): is the following still valid and usable? -->
```

Comment:
@theanalyst this is not valid, correct? cephadm will write the values to the ceph.conf, but if the user makes any changes, this will override, right?

Reply:
afaik yes, @jschmid1 isn't that correct wrt cephadm?